src/components/SearchBar/SearchBar.jsx

```diff
@@ -1,35 +1,25 @@
-import React, { Component } from 'react';
+import React, { PropTypes } from 'react';
 import { connect } from 'react-redux';
-import './SearchBar.css'
+import './SearchBar.css';
 // import PanelBar from '../PanelBar/PanelBar.jsx';
 
-export class SearchBarComponent extends Component {
-  constructor (props) {
-    super(props);
-
-    this.state = {};
-  }
-
-  render () {
-    return (
-      <div className="SearchBar">
-        <input type="text" placeholder="Search Chats..."/>
-      </div>
-    );
-  }
-}
-
-const mapStateToProps = state => ({
-
-});
-
-const mapDispatchToProps = dispatch => ({
-
-});
-
-const SearchBar = connect(
-  mapStateToProps,
-  mapDispatchToProps,
-)(SearchBarComponent);
-
-export default SearchBar;
+const SearchBar = props => (
+  <div className="SearchBar">
+    <input
+      type="text"
+      placeholder="Search Chats..."
+      value={props.query}
+      onChange={props.onQueryChange}
+    />
+  </div>
+);
+
+SearchBar.propTypes = {
+  query: PropTypes.string.isRequired,
+  onQueryChange: PropTypes.func.isRequired,
+};
+
+export default connect(
+  () => ({}),
+  () => ({}),
```

Comment:
Same as above, could we switch these back unless theres strong reasoning for this formatting?